components/list/test/list-item-drag-handle.test.js

```diff
@@ -0,0 +1,112 @@
+import { expect, fixture, html, oneEvent } from '@open-wc/testing';
+import { dragActions } from '../list-item-drag-handle.js';
+
+const keyCodes = Object.freeze({
+	END: { key: 'end', code: 35 },
+	HOME: { key: 'home', code: 36 },
+	UP: { key: 'up arrow', code: 38 },
+	DOWN: { key: 'down arrow', code: 40 },
+	SPACE: { key: 'space', code: 32 },
+	ENTER: { key: 'enter', code: 13 },
+	ESC: { key: 'escape', code: 27 },
+	TAB: { key: 'tab', code: 9 },
+	LEFT: { key: 'left arrow', code: 37 },
+	RIGHT: { key: 'right arrow', code: 39 }
+});
+
+describe('ListItemDragHandle', () => {
```

Comment:
We should include a constructor test for the drag handle - take a peek at one of the other components but I think it would basically be...
```
it('should construct d2l-list-item-drag-handle', () => {
	runConstructor('d2l-list-item-drag-handle');
});
```
The reason for this is - there are certain things that are not allowed (such as sprouting attributes on the element) in the constructor.  We've seen this come up a couple times and it's easily missed since it's not often that we construct elements using `document.createElement` in Lit elements, fixtures, or our demos.

Reply:
Cool, didn't know about this at all. Will add it.